fab/fabfile.py

```diff
@@ -145,6 +145,7 @@ def tag_commit(self):
             tagger={
                 'name': user.name,
                 'email': user.email,
+                'date': datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ'),
```

Comment:
this cued me off: https://developer.github.com/v3/git/tags/#create-a-tag-object. it was super finicky with the format. i'll try out `isoformat` -- it has to exactly match what they specify